Terminal.Gui/Views/TextView.cs

```diff
@@ -3760,9 +3760,12 @@ bool DeleteTextBackwards ()
 
 				historyText.Add (new List<List<Rune>> () { new List<Rune> (currentLine) }, CursorPosition);
 
-				currentLine.RemoveAt (currentColumn - 1);
-				if (wordWrap && wrapManager.RemoveAt (currentRow, currentColumn - 1)) {
-					wrapNeeded = true;
+				if (wordWrap) {
```

Comment:
Please hit CTRL-K, CTRL-D in VS2022 to reformat the source to match the coding standards.

Reply:
I applied the reformat but this is the result. Is there something specific that isn't matching?

Reply:
My bad. I got caught off by how the diff was showing. Sorry about that.